ui/src/main/Upload.svelte

```diff
@@ -13,6 +13,14 @@
     let upload: Promise<any>;
     let rejectionError: string;
     let parseProgress: Progress;
+    function handleUrlLoad() {
+        dragover = true;
+        var answer = prompt('Url of the pdf');
```

Comment:
I would just make this a `let` or `const`...
Also at some point I might try to convert this to a homegrown popup using the `Popup` component, but that isn't crucial!

Reply:
Nope, no specific reason to spawn a new proxy. I haven't found any reliable proxy. Hence I made one for testing. 
Please suggest to me if you find any existing reliable proxy.